hiera.yaml

```diff
@@ -4,12 +4,14 @@ defaults:
   datadir: data
   data_hash: yaml_data
 hierarchy:
+  - name: "Auditd Major Version"
+    path: "auditd-v%{facts.auditd_major_version}.yaml"
   - name: "OS + Release"
```

Comment:
Would it be more clear if this had an auditd directory (akin to the os directory) and a auditd/v3.yaml and auditd/v2.yaml?